src/com/bjorn/MakeChange/IMakeChange.java

```diff
@@ -0,0 +1,12 @@
+package com.bjorn.MakeChange;
+
+import java.util.HashMap;
+
+/**
+ * Created by bjornjohnson on 1/6/16.
+ */
+public interface IMakeChange {
+    void updateCoinAmountHashMap(String cointType, HashMap<String, Integer> hmap);
```

Comment:
Typo: `coinType`. And we already talked about naming your hashmap `hmap`. It might seem less important when defining an interface, but it's just as critical to use intention-revealing names here: I should be able to read over this interface and understand what each method does and each parameter means without needing help from the details of the implementation.